tests/unit/functions/test_phase_to_phase.py

```diff
@@ -51,14 +51,14 @@ def test_reactive_power(p: float, cos_phi: float, expected: float):
     [
```

Comment:
I'd like to parametrize the axis-height as well. These values were chosen to check the knee point, so let's use height=10.0 for them. And then we can add some more to test the axis height at a wind speed of let's say 20 km (nominal).